client/my-sites/earn/memberships/index.jsx

```diff
@@ -471,46 +476,66 @@ class MembershipsSection extends Component {
 
 	renderOnboarding( cta ) {
 		return (
-			<div className="memberships__onboarding-wrapper">
-				<div className="memberships__onboarding-column-info">
-					<div className="memberships__onboarding-header">
-						{ this.props.translate( 'Introducing Recurring Payments.' ) }
-					</div>
-					<p className="memberships__onboarding-paragraph">
-						{ this.props.translate(
-							'Start collecting subscription payments! Recurring Payments is a feature inside the block editor. When editing a post or a page you can insert a button that will allow you to collect paying subscribers.'
-						) }{ ' ' }
-						<ExternalLink
-							href="https://wordpress.com/support/recurring-payments-button/"
-							icon={ true }
-						>
-							{ this.props.translate( 'Learn more.' ) }
-						</ExternalLink>
-					</p>
-					{ cta }
-					<div className="memberships__onboarding-benefits">
-						<div>
-							<Gridicon size={ 18 } icon="checkmark" />
-							{ this.props.translate( 'Add multiple subscription options' ) }
-						</div>
-						<div>
-							<Gridicon size={ 18 } icon="checkmark" />
-							{ this.props.translate( 'Collect payments in 135 countries' ) }
-						</div>
-						<div>
-							<Gridicon size={ 18 } icon="checkmark" />
-							{ this.props.translate( 'Easily manage subscribers' ) }
+			<Card>
+				<div className="memberships__onboarding-wrapper">
+					<div className="memberships__onboarding-column-info">
+						<div className="memberships__onboarding-header">
+							{ this.props.translate( 'Accept payments on your website.' ) }
 						</div>
+						<p className="memberships__onboarding-paragraph">
+							{ this.props.translate(
+								'WordPress.com Payments makes it easy to sell physical and digital goods, accept donations, charge for in-person services, build subscription newsletters, and more.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **9** times:
`translate( 'WordPress.com Payments makes it easy to sell physical and digital goods, accept donations, charge for in-person services, build subscription newsletters, and more. One-time, monthly, and yearly credit and debit card payment options are supported.' )` *ES Score: 8*